mne/decoding/transformer.py

```diff
@@ -715,3 +717,164 @@ def transform(self, X):
         X = self.estimator.transform(X)
         X = np.reshape(X.T, [-1, n_epochs, n_times]).transpose([1, 0, 2])
         return X
+
+
+class Filterer(TransformerMixin):
+    """Estimator to filter RtEpochs
```

Comment:
What is RtEpochs?

It should at least filter a 3D numpy array, this is the decoding API we're ve decided.

The best is probably to add an axis parameter default to -1; this will filter along the time dimension for 2D array (e.g. raw, evoked), 3D (epochs) 4D (time freq power).